Dataset Collector #1, Chudinova Alla - 19FPL2

x-ae-a-12 opened the pull request.

x-ae-a-12 committed:
Minor changes for PR

dmitry-uraev committed:
[DEV] Updated due to main

dmitry-uraev commented on scrapper.py:
```diff
@@ -94,4 +94,8 @@ def validate_config(crawler_path):
+    response = requests.get('')
+    print('')
+    sleep (5) #import time first
+    response = request.get
```
let's start to implement code. I think it's a good time.

dmitry-uraev commented on target_score.txt:
```diff
@@ -1,5 +1,5 @@
 # Target score for scrapper.py:
-6
+8
```
Yes, that s good switch.

x-ae-a-12 replied:
> Yes, that s good switch.

No, I do not think so. It is eminent I'd better switch to 4 or 6 later.

dmitry-uraev commented on scrapper.py:
```diff
@@ -32,7 +32,7 @@ class Crawler:
     """
     def __init__(self, seed_urls: list, max_articles: int):
-        pass
+        self.seed_urls = []
```
initialize other params in the constructor

dmitry-uraev commented on scrapper.py:
```diff
@@ -94,4 +94,8 @@ def validate_config(crawler_path):
+    print('')
+    sleep (5) #import time first
+    response = request.get
+    #ILY
```
?

dmitry-uraev reviewed (changes requested):
It it time to start implementing something here. Or  implementing some essays may be?)

dmitry-uraev reviewed (commented):
It is right time to implement something. The deadline is coming.

x-ae-a-12 committed:
Scrapper commitment

I/ve done it with desktop app

x-ae-a-12 commented:
> It is right time to implement something. The deadline is coming.

ok

x-ae-a-12 committed:
Updated my scrapper.py

I wish I had green pr, mais on va le voir...

x-ae-a-12 committed:
Article.py update

Plaisir de l'article.py ne dure qu'un moment, chagrin du scrapper dure toute la vie

x-ae-a-12 committed:
I SHOULD PASS

Changed a line in my code in order to pass all (i hope) checks

dmitry-uraev reviewed (commented):
Nice. I like this realisation. We may add some styling corrections, but overall - super.

dmitry-uraev committed:
Merge remote-tracking branch 'upstream/main' into HEAD

x-ae-a-12 committed:
Updated raw_metadata_test

i need green pr

dmitry-uraev committed:
Merge remote-tracking branch 'upstream/main' into HEAD

x-ae-a-12 committed:
added a line to .pylintrc

x-ae-a-12 committed:
added pipeline realisation

x-ae-a-12 committed:
updated due to lint

x-ae-a-12 committed:
fix analysis

x-ae-a-12 committed:
added INTJ tag

dmitry-uraev commented:
Good! Not  best from the styling side, but acceptable for 8

x-ae-a-12 commented:
> Good! Not best from the styling side, but acceptable for 8

ok, thank you.
